blazarclient/v1/leases.py

```diff
@@ -32,11 +32,15 @@ def create(self, name, start, end, reservations, events, before_end=None):
         resp, body = self.request_manager.post('/leases', body=values)
         return body['lease']
 
-    def get(self, lease_id):
+    def get(self, lease_id, detail=False):
         """Describes lease specifications such as name, status and locked
         condition.
         """
         resp, body = self.request_manager.get('/leases/%s' % lease_id)
+        if detail and body['lease']:
```

Comment:
How easily could we parallelize this? This is now 4 requests, which might be fairly slow once deployed.